docs/src/recipes/shortest-path.asciidoc

```diff
@@ -48,6 +48,17 @@ course, it is possible for there to be more than one path in the graph of the sa
 length three), but this example is not considering that.
 <2> It might be interesting to know the path lengths for all paths between vertex "1" and "5".
 <3> Alternatively, one might wish to do a path length distribution over all the paths.
+<4> The preferred way to get a shortest path in OLAP is the `shortestPath()` step.
```

Comment:
hmm - `<4>` doesn't refer to anything in the code above. did you mean to add that callout?

Reply:
Ah, forgot to remove it, that's from when OLTP and OLAP were still mixed in one block.